hasjob/static/js/app.js

```diff
@@ -216,26 +194,45 @@ window.Hasjob.Filters = {
       selectedCurrency: window.Hasjob.Config.selectedFilters.currency,
       pay: window.Hasjob.Config.selectedFilters.pay,
       equity: window.Hasjob.Config.selectedFilters.equity,
-      sidebarOn: false
+      isMobile: $(window).width() < 768
     };
   },
   init: function(){
     var filters = this;
     var keywordTimeout;
-    var isSlidingMenu = $(window).width() < 768;
     var isFilterDropdownClosed = true;
     var filterMenuHeight = $('#hgnav').height() - $('#hg-sitenav').height();
     var pageScrollTimerId;
 
-    filters.ractive = new Ractive({
+    filters.dropdownMenu = new Ractive({
       el: 'job-filters-ractive-template',
       template: '#filters-ractive',
       data: this.getCurrentState(),
-      showSidebar: function() {
-        filters.ractive.set('sidebarOn', true);
+      show: function() {
+        filters.dropdownMenu.set('show', true);
+      },
+      hide: function() {
+        filters.dropdownMenu.set('show', false);
+      },
+      openDropDownMenu: function(event) {
```

Comment:
Is this and `closeDropDownMenu` really required? Since it's mostly just calling `show` (and `hide`).

Reply:
Fixed